pkg/controller.v2/controller.go

```diff
@@ -184,13 +189,14 @@ func NewTFJobController(
 
 	// Create new TFJobController.
 	tc := &TFJobController{
-		podControl:     realPodControl,
-		serviceControl: realServiceControl,
-		kubeClientSet:  kubeClientSet,
-		tfJobClientSet: tfJobClientSet,
-		expectations:   controller.NewControllerExpectations(),
-		workQueue:      workqueue.NewNamedRateLimitingQueue(workqueue.DefaultControllerRateLimiter(), tfv1alpha2.Plural),
-		recorder:       recorder,
+		podControl:           realPodControl,
+		serviceControl:       realServiceControl,
+		kubeClientSet:        kubeClientSet,
+		tfJobClientSet:       tfJobClientSet,
+		expectations:         controller.NewControllerExpectations(),
+		workQueue:            workqueue.NewNamedRateLimitingQueue(workqueue.DefaultControllerRateLimiter(), tfv1alpha2.Plural),
+		recorder:             recorder,
+		enableGangScheduling: option.EnableGangScheduling,
```

Comment:
I think we should define the field in TFJobControllerConfiguration and set it in config here https://github.com/kubeflow/tf-operator/pull/717/files#diff-adc8ad6b53f8e2be56aa2ba02afbcc2bR85

Reply:
Do you means set it like this

	// Create new TFJobController.
	tc := &TFJobController{
		config: TFJobControllerConfiguration{
			ReconcilerSyncLoopPeriod: metav1.Duration{Duration: 15 * time.Second},
			enableGangScheduling:     option.EnableGangScheduling,
		},
         }

Reply:
Yeah, I think it is better, WDYT